pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.5</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.dsrm</groupId>
+    <artifactId>dsrm-backend</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>dsrm-backend</name>
+    <description>dsrm-backend</description>
+    <properties>
+        <java.version>17</java.version>
```

Comment:
Add <snakeyaml.version>1.31</snakeyaml.version> . It is managed dependency by spring.